### PR TITLE
Feature/my assessments

### PR DESCRIPTION
**Pull request by ZiadAliKhalifa**
-Added a webpage to display all current user assessments
-Still might need to fix the looks of the page a little
-Still need to agree with the team on where to place the link to this page

### CODE REVIEW

File: src/api/myAssessmentsApi.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { handleAxiosResponse, handleError } from "./apiUtils";
+
+import axios from "axios";
+
+export function getAssessments(dataSourceId) {
+  debugger;
```
Comment (NadaBasha) [inline src/api/myAssessmentsApi.js]: can you please remove the debugger?

File: src/api/myAssessmentsApi.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { handleAxiosResponse, handleError } from "./apiUtils";
+
+import axios from "axios";
+
+export function getAssessments(dataSourceId) {
+  debugger;
+  const dataSourceIdd = dataSourceId;
+  return axios
+    .get("assessment/1231234", {
```
Comment (NadaBasha) [inline src/api/myAssessmentsApi.js]: shouldn't the datasourceid be used here?

File: src/redux/actions/actionTypes.js [hunk 3]
```diff
@@ -10,6 +10,8 @@ export const LOAD_TREE = "LOAD_TREE";
 export const LOAD_TREE_SUCCESS = "LOAD_TREE_SUCCESS";
 
 export const ASSESMENT = "ASSESMENT";
+export const LOAD_ASSESSMENTS = "LOAD_ASSESSMENTS";
+export const LOAD_ASSESSMENTS_SUCCESS = "LOAD_ASSESSMENTS_SUCCESS";
```
Comment (NadaBasha) [inline src/redux/actions/actionTypes.js]: is the success still needed?

File: src/views/MyAssessments/MyAssessments.jsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    marginBottom: "0px"
+  }
+};
+
+function MyAssessments({ getAssessmentGroups, currentDataSourceId }) {
+  const [assessments, setAssessments] = useState([]);
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    getAssessments(currentDataSourceId).then(assessments => {
```
Comment (NadaBasha) [inline src/views/MyAssessments/MyAssessments.jsx]: why is this using gteAssessments diect;y without thunk?
what is differnce between getting assesments and getting assessment groups?

File: src/api/myAssessmentsApi.js (hunk 1 above)
Comment (ZiadAliKhalifa) [reply src/api/myAssessmentsApi.js]: yes, I forgot to remove those.

File: src/api/myAssessmentsApi.js (hunk 2 above)
Comment (ZiadAliKhalifa) [reply src/api/myAssessmentsApi.js]: I remember changing this back, guess it was really late in the workday

File: src/redux/actions/actionTypes.js (hunk 3 above)
Comment (ZiadAliKhalifa) [reply src/redux/actions/actionTypes.js]: No, I will remove the success

File: src/views/MyAssessments/MyAssessments.jsx (hunk 4 above)
Comment (ZiadAliKhalifa) [reply src/views/MyAssessments/MyAssessments.jsx]: Get assessments groups is the action and get assessments is the Thunk. 
I changed them both to the action.

File: package.json [hunk 5]
```diff
@@ -7,8 +7,9 @@
     "@fortawesome/fontawesome-svg-core": "1.2.19",
     "@fortawesome/free-solid-svg-icons": "5.9.0",
     "@fortawesome/react-fontawesome": "0.1.4",
-    "@material-ui/core": "3.9.2",
+    "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "3.0.2",
+    "@material-ui/styles": "^4.2.0",
```
Comment (NadaBasha) [inline package.json]: Do we need the ^ ?

File: src/index.js [hunk 6]
```diff
@@ -13,6 +13,7 @@ import configureOkta from "configureOkta";
 import configureStore from "redux/configureStore";
 import { createBrowserHistory } from "history";
 import initialState from "redux/reducers/initialState";
+import MyAssessments from "views/MyAssessments/MyAssessments";
```
Comment (NadaBasha) [inline src/index.js]: do we need this import?

File: src/redux/actions/actionTypes.js [hunk 7]
```diff
@@ -10,6 +10,7 @@ export const LOAD_TREE = "LOAD_TREE";
 export const LOAD_TREE_SUCCESS = "LOAD_TREE_SUCCESS";
 
 export const ASSESMENT = "ASSESMENT";
+export const LOAD_ASSESSMENTS = "LOAD_ASSESSMENTS";
```
Comment (NadaBasha) [inline src/redux/actions/actionTypes.js]: We do not need this as well, right?

File: src/views/MyAssessments/MyAssessments.jsx [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+    getAssessmentGroups(currentDataSourceId).then(assessments => {
+      setAssessments(assessments);
+    });
+  }, [currentDataSourceId]);
+
+  useEffect(() => {
+    getAssessmentGroups(currentDataSourceId).then(res => {
+      setData(res);
+    });
+  }, []);
```
Comment (NadaBasha) [inline src/views/MyAssessments/MyAssessments.jsx]: do we need the 2 use effected?
what is the assessments state used for?

**Pull request merged**